<?xml version="1.0"?>
<launch>

    <arg name="config_file" default="$(find social_nav_metrics)/config/config_example.yaml"/>

    <node pkg="social_nav_metrics" type="metrics_recorder.py" name="metrics_recorder_node" output="screen">
        <rosparam command="load" file="$(arg config_file)"/>
    </node>

    <node pkg="social_nav_metrics" type="collision_counter" name="collision_counter_node" output="screen">
        <rosparam command="load" file="$(arg config_file)"/>
    </node>

    <node pkg="social_nav_metrics" type="acceleration_monitor.py" name="acceleration_monitor_node" output="screen">
        <rosparam command="load" file="$(arg config_file)"/>
    </node>
</launch>